lib/slapi/helpers/brain.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Slapi Brain Helper for Sinatra Extension Access
+class Slapi
+  def self.query_key(hash_name, key)
+    query = @brain.query_key(hash_name, key)
+    @logger.debug("Key retrieved for #{hash_name}")
```

Comment:
If you are only logging hash_name, you can move this above the query line and not have to repeat.

If you want to log the query at debug level then that should be added.